lib/jazzy/config.rb

```diff
@@ -23,8 +24,13 @@ class Config
     attr_accessor :version
     attr_accessor :min_acl
     attr_accessor :skip_undocumented
+    attr_accessor :podspec
```

Comment:
Config options generally have an exposed command line option. Why not add them for `docset_icon`, `docset_path` and `source_directory`?